packages/forklift-console-plugin/src/modules/Overview/dynamic-plugin.ts

```diff
@@ -37,4 +38,14 @@ export const extensions: EncodedExtension[] = [
       required: ['CAN_LIST_NS'],
     },
   } as EncodedExtension<HrefNavItem>,
+
+  {
+    type: 'console.context-provider',
+    properties: {
+      provider: { $codeRef: 'OverviewStateContextProvider.CreateOverviewStateContextProvider' },
+      useValueHook: {
+        $codeRef: 'OverviewStateContextProvider.useOverviewStateContextValue',
```

Comment:
```suggestion
        $codeRef: 'OverviewStateContextProvider.useOverviewContext',
```

I prefer a shorter name, in this case `state` is implicit from context and value is implicit from the word use